htmd/parameterization/fftype.py

```diff
@@ -3,31 +3,58 @@
 # Distributed under HTMD Software License Agreement
 # No redistribution in whole or part
 #
-
-import shutil
-import subprocess
 import os
+import logging
+import subprocess
+import unittest
+from yaml import load as yamlload
+from pickle import load as pickleload
 from tempfile import TemporaryDirectory
-from htmd.parameterization.readers import readPREPI, readFRCMOD, readRTF
+
 import numpy as np
 import parmed
-import logging
-import unittest
+
+from htmd.home import home
+from htmd.molecule.molecule import Molecule
+from htmd.parameterization.readers import readPREPI, readFRCMOD, readRTF
 
 logger = logging.getLogger(__name__)
 
 
 fftypemethods = ('CGenFF_2b6', 'GAFF', 'GAFF2')
 
 
-def listFftypemethods():
```

Comment:
Why delete this function?

Reply:
Dead code!

Reply:
Why is it dead? I created it on the last PR. It's an helper function.

Reply:
Not used, not tested, and just one line...

Reply:
It can be used by the user to get all available options like we do in htmd.builder. Anyway, let's stop this. It doesn't bother, there is nothing to test (it just prints the methods) so please restore it so we can finish the PR. We are a bit in a hurry.